.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,13 @@
+on: pull_request
+name: Auto Label Pull Request
+jobs:
+  pRLabelByFiles:
+    name: PR label by Files
+    runs-on: ubuntu-latest
```

Comment:
don't you think we should use a specific version as it is often recommended here?
`runs-on: ubuntu-18.04`